### PR TITLE
bug fix starting from 1 when initializing tree

### PR DESCRIPTION
**Pull request by sh-cha**
## Summary by CodeRabbit

- **New Features**
	- Enhanced migration command functionality to support additional versions and a new polling interval flag.
	- New migration functions introduced for improved tree entry management and asynchronous operations.

- **Bug Fixes**
	- Improved error handling and control flow in tree preparation and output management.

- **Refactor**
	- Simplified logic in the `Initialize` method for clearer control flow and error handling. 

- **Documentation**
	- Updated command descriptions for clarity on migration steps.

### CODE REVIEW

File: executor/db.go [hunk 1]
```diff
@@ -72,3 +77,139 @@ func Migration015(db types.DB) error {
...
+		return false, nil
+	})
+	if err != nil {
+		return err
+	}
+
+	nextSequence := uint64(1)
+	err = merkleDB.PrefixedIterate(merkletypes.WorkingTreeKey, nil, func(key, value []byte) (bool, error) {
+		if len(key) != len(merkletypes.WorkingTreeKey)+1+8 {
+			return false, nil
```
Comment (beer-1) [inline executor/db.go]: is it error case? if then good to print error or return error maybe?

File: executor/db.go [hunk 2]
```diff
@@ -72,3 +77,157 @@ func Migration015(db types.DB) error {
...
+				LeafCount:      workingTree.LeafCount,
+				ExtraData:      data,
+			}
+
+			finalizedTreeBz, err := json.Marshal(finalizedTreeInfo)
+			if err != nil {
+				return true, err
+			}
+
+			err = merkleDB.Set(finalizedTreeInfo.Key(), finalizedTreeBz)
```
Comment (beer-1) [inline executor/db.go:164]: don't need to use PrefixedKey?

File: executor/db.go [hunk 3]
```diff
@@ -72,3 +77,157 @@ func Migration015(db types.DB) error {
...
+		}
+
+		var extraData executortypes.TreeExtraData
+		err = json.Unmarshal(tree.ExtraData, &extraData)
+		if err != nil {
+			return true, err
+		}
+
+		if extraData.BlockHash != nil {
+			return false, nil
```
Comment (beer-1) [inline executor/db.go:199]: Is this error case? then seems good to print warning

File: executor/db.go [hunk 4]
```diff
@@ -72,3 +77,157 @@ func Migration015(db types.DB) error {
...
+		for {
+			select {
+			case <-ctx.Done():
+				return true, ctx.Err()
+			case <-timer.C:
+			}
+			height := extraData.BlockNumber + 1
+			header, err := rpcClient.Header(ctx, &height)
+			if err != nil {
+				continue
```
Comment (beer-1) [inline executor/db.go:214]: good to give progress or error logs here

File: executor/db.go (hunk 2 above)
Comment (sh-cha) [reply executor/db.go:164]: it doesn't need it

**Review by beer-1 — APPROVED** [review]
LGTM

**Pull request merged**